app/components/UI/TransactionElement/index.js

```diff
@@ -5,6 +5,7 @@ import { fontStyles } from '../../../styles/common';
 import FAIcon from 'react-native-vector-icons/FontAwesome';
 import { strings } from '../../../../locales/i18n';
 import { toDateFormat } from '../../../util/date';
+import TransactionDetails from './TransactionDetails';
```

Comment:
Do you still need TransactionDetails here ? If not then we need to remove it

Reply:
this is a different transaction details, it was created by MMM team. I believe they need it still for the previous modal popout